modules/nw-network-observability-operator.adoc

```diff
@@ -0,0 +1,64 @@
+/ Module included in the following assemblies:
+// * networking/network_observability/understanding-network-observability-operator.adoc
+
+:_content-type: PROCEDURE
+[id="nw-network-observability-operator_{context}"]
+= Network Observability Operator
+
+When you deploy flow collector, the Operator provides dashboards, metrics, and keeps flows accessible in a queryable log store, Grafana Loki. When a FlowCollector is deployed, new dashboards are available in the Console.
```

Comment:
```suggestion
When you deploy flow collector, the Operator provides dashboards, metrics, and keeps flows accessible in a queryable log store, Grafana Loki. When a FlowCollector is deployed, new dashboards are available in the Console.
```
We should probably standardize `flow collector` vs `FlowCollector`? Not sure which is more appropriate